### DISCUSSION TITLE
Aquifers

### DISCUSSION THREAD

**Discussion by masoudsouri** [iiasa/CWatM#160] [General]
Dear CWatM Community.
How does CWatM identify the cells that have aquifers beneath them? Additionally, is the recharge provided by CWatM to the groundwater module only associated with the cells located directly above aquifers, or does it apply to all cells within the watershed? Please explain.

**Comment by dof1985** [comment]
Dear @masoudsouri 

Thank you for your question. As a fully-distributed hydrological model, CWatM maintain water flows and balance for each grid cell. It simulates a channel, soil, and an aquifer (saturated underground soil storage). Recharge simulation is based on the unsaturated conductivity of the third soil layer (based on the Van Genuchten model), and only feed the 'aquifer' in the same grid cell.

A more realistic model of GW is possible with the coupled CWatM-Modflow. In this case recharge still feed into the grid cell 'aquifer', but lateral flows in the sub-surface saturated layer is enabled via Modflow. 

You can further read about the basic CWatM here: https://gmd.copernicus.org/articles/13/3267/2020/
and about the coupling with Modflow here: https://gmd.copernicus.org/articles/15/7099/2022/

Good luck
